test/job/test.sh

```
@@ -23,41 +27,46 @@ export service_account_name=${SERVICE_ACCOUNT_NAME:-"default"}
-envsubst '${image} ${agent_namespace} ${consumer_name} ${service_account_name}' < ${PWD}/test/e2e/istio/job.yaml.template | kubectl --kubeconfig=$server_kubeconfig apply -f -
+# deploy the test job
+envsubst '${job_name} ${job_cmd} ${image} ${agent_namespace} ${consumer_name} ${service_account_name}' < ${PWD}/test/job/job.yaml.template | kubectl --kubeconfig=$server_kubeconfig apply -f -
 
 sleep 5
 
 # Wait for pod to exist and be running
-echo "Waiting for job maestro-e2e-tests to start..."
+echo "Waiting for job ${job_name} to start..."
 while true; do
-  PHASE=$(kubectl --kubeconfig=$server_kubeconfig -n clusters-service get pods -l app=maestro-e2e -o jsonpath='{.items[0].status.phase}' 2>/dev/null)
+  PHASE=$(kubectl --kubeconfig=$server_kubeconfig -n clusters-service get pods -l name={job_name} -o jsonpath='{.items[0].status.phase}' 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)